### ISSUE TITLE
Has any date to update this package for release for .NET 8

### ISSUE THREAD

**Issue by GUSTAVPEREIRA** [yartat/MP-MediaInfo#51]

**Comment by MichaelHochriegl** [comment]
Unfortunately this lib is not actively maintained currently, so I guess a PR would be pointless right now.
If someone knows another good MediaInfo lib for dotnet, please tell me, as this is crucial for the application I'm building.

I tried to get my own going, but I don't know how to properly provide the native dlls that will be needed.

**Comment by Dunge** [comment]
Referencing the nuget from a NET8 application is still possible.

You get a warning about RID, but it works.

On Windows it works directly with no changes.
On a NET8 Linux Docker image, I had to hack my way around it and add this line to my Dockerfile:

> COPY --from=build /app/publish/runtimes/debian.11-x64/native/libmediainfo.so ./runtimes/linux-x64/native/

**Comment by GUSTAVPEREIRA** [comment]
Yeah, I tried running this on Linux Fedora and Ubuntu, but as Michael said, I believe it's better to create another project and start developing with it. From my point of view, this is deprecated. But thanks, everyone, for helping us.
